Resources/Prototypes/Entities/Objects/Tools/t-ray.yml

```
@@ -2,6 +2,7 @@
   name: t-ray scanner
   parent: BaseItem
   id: trayScanner
+  description: A high-tech scanning device that uses Terahertz Radiation to detect subfloor objects.
```

> Should probably be reworded to say it detects under-floor objects somehow, "objects" feels pretty vauge and potentially misleading.

The word "infrastructure" comes to mind.
```suggestion
  description: A high-tech scanning device that uses Terahertz Radiation to detect subfloor infrastructure.
```